scripts/ws-hoard.sh

```
@@ -0,0 +1,320 @@
+        cd "$target"
+        git init -q
+        git add .
+        git -c user.email="hoard@local" -c user.name="hoard-init" \
+            commit -q -m "Initial commit (${template} hoard for ${who})"
+    )
+
+    echo ""
+    echo "Hoard initialized: $target"
+    echo ""
+    echo "Push to your own remote when ready, e.g.:"
+    echo "  gh repo create ${who}/${template}-${who} --private --source=${target#$ROOT_DIR/}"
```

Shellcheck (0.11.0)

[info] Expansions inside ${..} need to be quoted separately, otherwise they match as patterns.

(SC2295)